### ISSUE TITLE
explore ways to sample

### ISSUE THREAD

**Issue by troettge** [troettge/Research-NORRN#24]

**Comment by troettge** [comment]
explore possible ways to access as many articles as possible. Probably easiest to follow the [publication guidelines](https://www.fhi.no/globalassets/dokumenterfiler/studier/den-norske-mor-far-og-barn--undersokelsenmoba/info-til-forskere/publication_guidelines_moba.pdf) and use web of science or gscholar

**Comment by timo-b-roettger** [comment]
Possible strategies following the guidelines:
- title --> not useful. Most articles do not refer to dataset in title
- web of science all fields (including abstract, title, etc.): search string "Norwegian Mother Father and Child Cohort" yields **183 hits**
- web of science all fields (including abstract, title, etc.): search string "Medical Birth Registry of Norway" yields **606 hits** --> promising
- references: GScholar citing Magnus et al. (2016) yields 639 hits. --> Gscholar might be less curated than web of science, so I suggest going for the 606 hits above

**Comment by Tamara-Kalan** [comment]
Agree- 606 hits sound promising. We can then have a look and decide if we need to change the search strategy.

Ps. Saw this article now and it looks like the one I was invited to be involved. This is from their language sub-study  https://onlinelibrary.wiley.com/doi/10.1111/1460-6984.12861 :-)
